Prusament_petg_175.xml.fdm_material

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Prusament 1.75mm PLA Galaxy Black profile. The data in this file may not be correct for your specific machine.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material" version="1.3">
+    <metadata>
+        <name>
+            <brand>Prusament</brand>
+            <material>PLA</material>
+            <color>Galaxy Black</color>
+        </name>
+        <GUID>a11bd33e-3ff1-4a5f-aea7-365156d6e427</GUID>
+        <version>1</version>
+        <color_code>#000000</color_code>
+        <description>Prusament PETG is our own in-house made filament. The whole manufacturing process is closely monitored and tested - we guarantee ±0.02mm precision and highly-consistent colors.</description>
+        <adhesion_info>Print on bare glass. Use tape for cold build plates.</adhesion_info>
+    </metadata>
+    
+    <properties>
+        <density>1.38</density>
```

Comment:
Density of PETG is about 1.24-1.27. 
Check how it is defined in Prusa Slicer for those materials:
https://github.com/prusa3d/PrusaSlicer/blob/master/resources/profiles/PrusaResearch.ini